packages/icons/scripts/build-icons/writeBundle.js

```diff
@@ -1,25 +1,50 @@
 const fs = require("fs");
 const capitalizeCamelCase = require('./capitalizeCamelCase');
 
+function stylize(svgName) {
+  return 'props => (\n' +
+    '  <ThemeContext.Consumer>\n' +
+    '    {({ resolvedRoles }) => {\n' +
+    '      const propsClone = Object.assign({}, props, { className: cx(css(stylesheet(resolvedRoles)), props.className) });\n' +
```

Comment:
So with `Icons` we allow the user to pass a `className` in?

Reply:
Yes and it'll get merged in with the Icon stylesheet. This was necessary anyway because consumers of icons would typically add additional classes to customize the styling of the icon for the component (see button, dropdown, etc)

Reply:
We may need to reconcile it eventually. You could try merging my branch in and see if it affects the layout.

Reply:
Yah, I thought w/ themeable components we were removing css class names as hooks for people to use.

Reply:
In further thinking about it, I think we probably want to do something like this in other components as well in case users pass in a `className` prop.